src/Umbraco.Core/Models/Membership/IUserGroup.cs

```diff
@@ -22,6 +22,11 @@ public interface IUserGroup : IEntity, IRememberBeingDirty
     /// </summary>
     string? Name { get; set; }
 
+    /// <summary>
+    ///    The description
+    /// </summary>
+    string? Description { get; set; }
```

Comment:
As this is an interface, adding a new property will be considered a breaking change.  So you should add a default implementation to this property, like this:

```
string? Description
{
    get => null;
    set { }
}
```